### ISSUE TITLE
Unable to select 64 bit version for Delphi 10.2 Tokyo

### ISSUE THREAD

**Issue by SteveDIY** [DelphiPraxis/The-Drag-and-Drop-Component-Suite-for-Delphi#28]
I am unable to select 64 bit windows target platform when I open dclDragDropDR102T.dpk or DragDropDR102T or DragDropDR102TGroup.groupproj in Delphi Tokyo. It allows only 32bit target platform.

**Comment by landrix** [comment]
DesignTime-packages don't has a 64 bit version. Only the RunTime-package needs to be compiled for 64 bit.

**Comment by SteveDIY** [comment]
You mean DragDropDR102T.dpk? Opening this DPK in Delphi 10.2 Tokyo does not enable 64 bit target platform.

**Comment by SteveDIY** [comment]
Partial screenshot, if that helps:
![image](https://user-images.githubusercontent.com/38472442/38888377-7146cbac-4249-11e8-8be0-76f446565d48.png)

**Comment by landrix** [comment]
yes, because it is a design time package for IDE only, and IDE don't support 64 bit themselfe.

**Comment by SteveDIY** [comment]
I am sorry for taking up your valuable time, but Im still confused about "Only the RunTime-package needs to be compiled for 64 bit". How do I do that? What project/dpk file from the download should I open in tokyo IDE so that I can compile for 64 bit?

**Comment by SteveDIY** [comment]
I was checking the source code of DragDropDR**101B.dproj** in 10.1 Berlin project. I found it has following lines

            <Platforms>
                <Platform value="Win32">True</Platform>
                <Platform value="Win64">True</Platform>
            </Platforms>

While DragDropDR**102T.dproj** has only

            <Platforms>
                <Platform value="Android">False</Platform>
                <Platform value="iOSDevice32">False</Platform>
                <Platform value="iOSSimulator">False</Platform>
                <Platform value="Linux64">True</Platform>
                <Platform value="Win32">True</Platform>
</Platforms>

Since it does not have win64 entry, could it be the reason I'm not able to select 64 bit version for compilation?

**Comment by SteveDIY** [comment]
Update: I could make it compile using following 2 changes:

(1) In DragDropDR102T.dproj, I manually added <Platform value="Win64">True</Platform> entry as described above.
(2) When I opened this project in 10.2 Tokyo IDE, I changed the project option "Unit output directory" to ".\$(Platform)\$(Config)"

Upon compile, DCU's were created in appropriate platform directories. And the BPL, too was created in appropriate platform directory (Win64 or Win32) at C:\Users\Public\Documents\Embarcadero\Studio\19.0\Bpl.

Hope this helps others, too.
